cmd/restic/exclude_test.go

```diff
@@ -189,3 +189,111 @@ func TestMultipleIsExcludedByFile(t *testing.T) {
 		}
 	}
 }
+
+func TestParseSizeStr(t *testing.T) {
+	sizeStrTests := []struct {
+		in       string
+		expected int64
+	}{
+		{"1024", 1024},
+		{"1024b", 1024},
+		{"1024B", 1024},
+		{"1k", 1024},
+		{"100k", 102400},
+		{"100K", 102400},
+		{"10M", 10485760},
+		{"100m", 104857600},
+	}
+
+	for _, tt := range sizeStrTests {
+		actual, err := parseSizeStr(tt.in)
+		test.OK(t, err)
+
+		if actual != tt.expected {
+			t.Errorf("parseSizeStr(%s) = %d; expected %d", tt.in, actual, tt.expected)
+		}
+	}
+}
+
+// TestMultipleIsExcludedByFileSize is for testing that multiple instances of
+// the --exclude-if-present parameter (or the shortcut --exclude-caches do not
```

Comment:
These comments look like copy&paste from another test. Please adjust them to match this test. And I think it should be called `TestIsExcludedByFileSize`, at least I don't see multiple excludes here.